iOS-Settings/Sources/App/DetailViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DetailViewController.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 03.03.2023.
+//
+
+import UIKit
+import SnapKit
+
+final class DetailViewController: UIViewController {
+
+    private let setting: Setting?
+
+    // MARK: - Initializers
+
+    init(setting: Setting?) {
+        self.setting = setting
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError()
```

Comment:
fatalError() should be descriptive. At least like this fatalError("init(coder:) has not been implemented")